skopt/optimizer/optimizer.py

```diff
@@ -176,7 +186,101 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
                              "'sampling', got {0}".format(acq_optimizer))
         self.acq_optimizer = acq_optimizer
 
-    def ask(self):
+    def copy(self, copy_random_state=True):
+        """Creates a shallow copy of optimizer."""
+
+        params = {
+            'dimensions':self.space.dimensions,
```

Comment:
Yup, `pep8` is my current weakness. I will use heavily `pep8` package, so these should go away in nearby future.